alacritty_terminal/src/term/mod.rs

```diff
@@ -746,8 +758,9 @@ pub struct Term<T> {
     /// Current forward and backward buffer search regexes.
     regex_search: Option<RegexSearch>,
 
-    /// Information about window dimensions.
-    size: SizeInfo,
+    /// Information about cell dimensions.
+    cell_width: f32,
+    cell_height: f32,
```

Comment:
There's no point in using f32 for those.

Reply:
That's just the format they're in ***everywhere***. Do you suggest changing this everywhere in this PR?

Reply:
I mean that you convert them to usize or whatever you want, when you set those from resize, since you know that those are not fractional, so it'll reduce conversions later.